### DISCUSSION TITLE
Adding plugin to HomeKit

### DISCUSSION THREAD

**Discussion by muchuk** [AlexGustafsson/homebridge-wol#231] [Questions]
### Backstory

Please could someone kindly talk me through the setup of the WOL plugin? I've read over the wiki, and it doesn't solve my issue.

I can get WOL to work with purpose made iOS app which shows the magic packet is being received and is working. So the BIOS and network adapters have been configured properly.

### Issue

I've attached below the device config. I've taken out the IP and MAC addresses, but they're correctly entered.

The issue I have is with the iOS Homekit App. When I've added the plugin to HOOBs I then attempt to add the accessory to Home app.  The Home app can find the Wol Bridge listed as an available accessory. But when I select it, there's a pop-up requesting the Setup Code "enter the code found on the accessory, box or instructions." As there's no accessory set-up code, I cannot add the button to Homekit.

Any assistance would be greatly appreciated.

### Environment

```Markdown
- os and os version:
- node version:
- homebridge version: Hoobs v4.2
- homebridge-wol version:
```

### Configuration

"accessories": [
{
"name": "PC",
"manufacturer": "homebridge-wol",
"model": "NetworkDevice",
"host": "IP ADDRESS", 
"pingInterval": 2,
"pingsToChange": 5,
"pingTimeout": 1,
"pingCommandTimeout": 0,
"mac": "MAC ADDRESS",
"startCommandTimeout": 0,
"wakeGraceTime": 45,
"wakeCommandTimeout": 0,
"shutdownGraceTime": 15,
"shutdownCommandTimeout": 0,
"accessory": "NetworkDevice",
"serialNumber": "225"

### Logs

```Markdown
When I follow these steps:
1. Make sure `logLevel` is set to `Debug` on all `NetworkDevice` accessories
2. Run `homebridge`
3. Let it run for a couple of minutes
4. Try to turn a device on or off using the plugin
5. Wait one minute
6. Repeat step 4

I get the following log:

Please paste your log here
```
```


### Notes

_No response_

**Comment by AlexGustafsson** [comment]
> As there's no accessory set-up code, I cannot add the button to Homekit.

This set up code can be find in your config or in the Homebridge logs. Note that this is not an issue with this plugin, rather Homebridge itself. If you can't find the code, please open up a discussion over at the Homebridge repository to get it sorted. Closing.
